src/utils/html.js

```diff
@@ -1,9 +1,5 @@
 const stripTagsRegex = /(<([^>]+)>)/gi
 
-export function stripTags(text = '') {
-  return String(text).replace(stripTagsRegex, '')
-}
+export const stripTags = (text = '') => String(text).replace(stripTagsRegex, '')
 
-export function htmlOrText(innerHTML, textContent) {
-  return innerHTML ? { innerHTML } : { textContent }
-}
+export const htmlOrText = (innerHTML, textContent) => (innerHTML ? { innerHTML } : { textContent })
```

Comment:
This helper I would like to tweak a bit, as it kind of assumes that on or the other argument is present, but doesn't downgrade nicely when the second argument is undefined (which can/will be handy in some situations where we only want domProps for innerHTML and not innerText)

Reply:
Ok, feel free to apply you changes to this PR!

Reply:
Tweaked... it now fallbacks to an empty object if both `innerHTML` and `textContent` are falsey.